demonstrations/gqe_training.metadata.json

```
@@ -0,0 +1,38 @@
+{
+    "title": "Generative quantum eigensolver training using Pennylane data",
+    "authors": [
+        {
+            "id": "joseph_bunao"
```

Do you have a PennyLane Profile? You should make one if you don't have one already. :) Then you should use this:
```suggestion
            "username": "your_username"
```